packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts

```diff
@@ -139,13 +139,27 @@ export default function useSource(noteBody: string, noteMarkupLanguage: number,
 			js.push('}');
 			js.push('true;');
 
+			// iOS doesn't automatically adjust the WebView's font size to match users'
+			// accessibility settings. To do this, we need to tell it to match the system font.
+			// See https://github.com/ionic-team/capacitor/issues/2748#issuecomment-612923135
+			const iOSSpecificCss = `
+				@media screen {
+					:root body {
+						font: -apple-system-body;
```

Comment:
Won't that conflict with the font family defined in the settings?

Reply:
I'm not seeing a way to change the NoteBodyViewer's font (on mobile)...

If there is, adding injected JavaScript that removes the fontFamily but keeps the fontSize could work...

Reply:
Oh that's right, the font can only be changed for the editor. So I think this is fine then